src/engine/enginemaster.cpp

```diff
@@ -97,6 +100,13 @@ EngineMaster::EngineMaster(UserSettingsPointer pConfig,
 
     m_pMasterDelay = new EngineDelay(group, ConfigKey(group, "delay"));
     m_pHeadDelay = new EngineDelay(group, ConfigKey(group, "headDelay"));
+    m_pInputLatencyCompensationDelay = new EngineDelay(group,
+        ConfigKey(group, "inputLatency"));
+    m_pInputLatencyCompensationHeadphonesDelay = new EngineDelay(group,
+        ConfigKey(group, "headMixDelay"));
```

Comment:
>> Recording: In this case, the user does not hear the master and the talk over channel.
Huh? 

> Why would the user not want to hear these?

Because he would hear his own voice twice. First direct air feedback, second the delayed signal thou the head phone. You cannot speak good under that circumstances.

> m_pInputLatencyCompensationHeadphonesDelay only delays the PFL mix going to the headphones. The talkover mix going to the headphones is not delayed.

The use case is only relevant, if the speaking person is not the headphone listening person. I cannot think how this will work.
So for now: headphone, no talkover.
 
> I think you are confusing the latency compensation with the delays compensating for speaker placement. They are separate issues and the delays need to remain separate.

That's my point. Separate issues. I just wanted to point out, that you cannot do both at the same time, because it is just the opposite.